ee/components/LicenseBanner.tsx

```diff
@@ -89,8 +89,8 @@ export default function LicenseBanner() {
           <li>{t("cannot_repackage_codebase")}</li>
           <li>
             {t("acquire_license")}:{" "}
-            <a className="text-blue-500 underline" href="mailto:license@cal.com">
-              license@cal.com
+            <a className="text-blue-500 underline" href="mailto:talent-support@theskills.com">
+              talent-support@theskills.com
```

Comment:
So this actually needs to be changed back to `license@cal.com`. One of the few rate instances where we'll keep the cal.com reference, since it deals with licensing.